frontend/test/resolution/testScopeTypes.cpp

```diff
@@ -215,7 +217,7 @@ static void testBorrowIds() {
     ids.appendIdAndFlags(x, Decl::PUBLIC,
                          /* method */ false, /* parenful */ false);
     IdAndFlags::Flags f = 0;
-    IdAndFlags::Flags e = pub;
+    auto e = FlagSet::singleton(pub);
     auto foundIds = ids.borrow(f, e);
     assert(!foundIds.hasValue());
```

Comment:
Can you add some testing of the new FlagSet directly including the disjunction cases?